internal/runtime/client/client_test.go

```diff
@@ -1150,10 +1153,7 @@ func response(status runtimehooksv1.ResponseStatus) testServerResponse {
 	}
 }
 
-func createSecureTestServer(g *WithT, server testServerConfig) *httptest.Server {
-	l, err := net.Listen("tcp", server.hostPort)
-	g.Expect(err).NotTo(HaveOccurred())
-
+func createSecureTestServer(server testServerConfig) *httptest.Server { // FIXME
```

Comment:
What's the FIXME for?

Reply:
Dropping the `g` parameter which I did :)